keps/sig-testing/20190118-breaking-apart-the-kubernetes-test-tarball.md

```diff
@@ -0,0 +1,274 @@
+---
+kep-number: ?
+title: Breaking apart the Kubernetes test tarball
+authors:
+  - "@ixdy"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+reviewers:
+  - TBD
+approvers:
+  - "@spiffxp"
```

Comment:
@tpepper can you be an approver as a representative of sig-release's release-engineering subproject

Reply:
Definitely!

Reply:
added, PTAL, thanks!